### ISSUE TITLE
Throw errors when load a single-bundle app in browser

### ISSUE THREAD

**Issue by xiaohulu** [dojo/cli-build-app#338]
**Bug** 

Package Version: 6.0.8

**Code**

1. run `dojo build --mode dist --single-bundle`
2. cd `output/dist` and run `serve`
3. access `http://localhost:5000` in browser

**Expected behavior:**

Load and show index.html correctly.

**Actual behavior:**

Throw errors

```text
Uncaught ReferenceError: __DOJO_SCOPE is not defined
    at Object.<anonymous> (common.js:4)
    at n (bootstrap:19)
    at Object.<anonymous> (sync.js:2)
    at n (bootstrap:19)
    at Object.<anonymous> (App.m.css:2)
    at n (bootstrap:19)
    at bootstrap:83
    at bootstrap:83
    at universalModuleDefinition:9
    at universalModuleDefinition:1
```

**Comment by xiaohulu** [comment]
@matt-gadd Thanks

**Comment by agubler** [comment]
Hey @xiaohulu this has been fixed in the `6.0.9` release

**Comment by xiaohulu** [comment]
Ok, Thanks!
